util/network-devp2p/src/ip_utils.rs

```diff
@@ -236,16 +236,15 @@ mod getinterfaces {
 				let sa: *const sockaddr_in6 = sa as *const sockaddr_in6;
 				let sa = & unsafe { *sa };
 				let (addr, port) = (sa.sin6_addr.s6_addr, sa.sin6_port);
```

Comment:
what is the type of `addr` here?

Reply:
It is u32 but I saw that this was actually from the `std::net`  and it actually implements `from trait` with this so I changed it! Makes the code more readable IMO!